ros2_control_demo_hardware/src/rrbot_system_multi_interface.cpp

```diff
@@ -309,9 +311,9 @@ return_type RRBotSystemMultiInterfaceHardware::write()
     // Simulate sending commands to the hardware
     RCLCPP_INFO(
       rclcpp::get_logger("RRBotSystemMultiInterfaceHardware"),
-      "Got the commands pos: %.5f, vel: %.5f, acc: %.5f for joint %d, control_lvl: %d",
+      "Got the commands pos: %.5f, vel: %.5f, acc: %.5f for joint %zu, control_lvl: %hhu",
       hw_commands_positions_[i], hw_commands_velocities_[i], hw_commands_accelerations_[i], i,
-      control_lvl_[i]);
+      static_cast<std::uint8_t>(control_lvl_[i]));
```

Comment:
Is there any other solution then adding static_cast? This doesn't look nice in the code.